enhancements/installer/rhel8_worker_support.md

```diff
@@ -0,0 +1,252 @@
+---
+title: rhel-8-server-worker-infra-node-support
+authors:
+  - "@mtnbikenc"
+reviewers:
+  - "@staebler"
+approvers:
+  - "@derekwaynecarr"
+creation-date: 2021-05-13
+last-updated: 2021-05-13
+status: provisional
+see-also:
+  - "https://issues.redhat.com/browse/CORS-1650"
+replaces:
+  - n/a
+superseded-by:
+  - n/a
+---
+
+# RHEL 8 Server Worker/Infra Node Support
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Operational readiness criteria is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Summary
+
+This enhancement proposes to add RHEL 8 Server support for deploying worker/infra
+nodes with [openshift-ansible] RHEL 8 nodes will be deployed and upgraded using
+the same Ansible playbook process however, RPM content will be delivered using
+an image in the release package.
+
+## Motivation
+
+Customers desire to deploy RHEL 8 nodes and using containerized content will provide
```

Comment:
Is the use case for deploying RHEL 8 nodes from scratch?  Or would customers get the "base RHEL 8" content onto a host and then install the OCP specific RPMS(i.e.`cri-o`, `openshift-hyperkube`, etc)?

Reply:
The intent is that we are only installing the packages necessary for OCP from the container.  This alleviates the need of switching repo configuration for OCP upgrades between .y releases.  The rest of the OS content would come through the customer's existing OS management processes.

Reply:
`Customers desire to add existing subscribed RHEL 8 hosts to an OpenShift 4.9+ cluster as workers.` or something along those lines to address Micah's question, subscribed being important part because we need to be able to assume base channels are enabled to fulfill any deps.